jbpm-installer/src/test/java/org/jbpm/persistence/scripts/util/SQLScriptUtil.java

```diff
@@ -53,38 +66,41 @@
      */
     public static List<String> getCommandsFromScript(final File script, final DatabaseType databaseType)
             throws IOException {
+        String delimiter = getDelimiterByFileAndDatabase(script, databaseType);
+        String escapedDelimiter = DELIMITER_STANDARD.equals(delimiter) ? "\\" + DELIMITER_STANDARD : delimiter;
+        
         final List<String> scriptLines = FileUtils.readLines(script, StandardCharsets.UTF_8);
         final List<String> foundCommands = new ArrayList<String>();
         final StringBuilder command = new StringBuilder();
         int pgBlockCounter = 0;
         for (String line : scriptLines) {
             // Ignore comments.
             final String trimmedLine = line.trim();
-            if ("".equals(trimmedLine) || trimmedLine.startsWith("--") || trimmedLine.startsWith("#")) {
+            if ("".equals(trimmedLine) || trimmedLine.startsWith("--") || trimmedLine.startsWith("#")
+                    || (trimmedLine.startsWith("/*"))) {
                 continue;
             }
             
-            if (databaseType == DatabaseType.POSTGRESQL && trimmedLine.matches(DOLLAR_QUOTED_BLOCK)) {
+            if (databaseType == POSTGRESQL && trimmedLine.matches(DOLLAR_QUOTED_BLOCK)) {
                 pgBlockCounter++;
             }
             
             // If the whole line is a delimiter -> add buffered command to found commands.
-            if (trimmedLine.equals(DELIMITER_STANDARD)
-                    || ((databaseType == DatabaseType.SQLSERVER || databaseType == DatabaseType.SQLSERVER2008)
-                            && trimmedLine.equals(DELIMITER_MSSQL))) {
+            if (trimmedLine.equals(delimiter)) {
                 if (!"".equals(command.toString())) {
                     foundCommands.add(command.toString());
                     command.setLength(0);
                     command.trimToSize();
                 }
             }
-            // Split line by delimiter.
-            if ((trimmedLine.contains(DELIMITER_STANDARD) && databaseType != DatabaseType.POSTGRESQL)
-                || trimmedLine.contains(DELIMITER_STANDARD) && databaseType == DatabaseType.POSTGRESQL && isEven(pgBlockCounter)) {
-                extractCommandsFromLine(trimmedLine, "\\" + DELIMITER_STANDARD, command, foundCommands);
-            } else if ((databaseType == DatabaseType.SQLSERVER || databaseType == DatabaseType.SQLSERVER2008)
-                    && trimmedLine.contains(DELIMITER_MSSQL)) {
-                extractCommandsFromLine(trimmedLine, DELIMITER_MSSQL, command, foundCommands);
+            
+            if (databaseType == SYBASE && !trimmedLine.startsWith(delimiter) && trimmedLine.contains(delimiter)){
```

Comment:
So `go` delimiter cannot be at the beginning of the line? Why is this step necessary? I would at least omit the `!trimmedLine.startsWith(delimiter)` since it can happen that there will be something at the beginning of the line which will start with `go`, e.g. column definition with column name `goal`.

Or maybe we can include `&& databaseType != SYBASE` in the next else if. e.g.

```
else if ((trimmedLine.contains(delimiter) && databaseType != POSTGRESQL && databaseType != SYBASE)
                || trimmedLine.contains(delimiter) && databaseType == POSTGRESQL && isEven(pgBlockCounter))
```

So the case where there is `go` somewhere in the middle of the line will be covered by the next `else` wdyt?

Reply:
We need to control that the delimiter breaks the line, but in the case of sybase (go) may be part of any other string (like constraint hashes, or column, etc).
Following your recommendation, I've finally considered a regex (delimiter+$, where $ is the end-of-line) to match and break the line
```
if ((trimmedLine.contains(delimiter) && databaseType != POSTGRESQL && databaseType != SYBASE)
|| trimmedLine.contains(delimiter) && databaseType == POSTGRESQL && isEven(pgBlockCounter)
|| trimmedLine.matches(delimiter+REGEX_EOL) && databaseType == SYBASE) 
```